chat.js

```diff
@@ -389,6 +397,9 @@ class CommandContext {
 		if (room.filterCaps && user.name.match(/[A-Z\s]{6,}/)) {
 			return this.errorReply(`Your username contains too many capital letters, which this room doesn't allow.`);
 		}
+        if (room.filterEmojis && emojiRegex.exec(user.name)) {
```

Comment:
I copied them from emoji list on Wikipedia, that could be why.

©️©

Wikipedia list on https://en.wikipedia.org/wiki/Emoji#Unicode_blocks lists U+00A9, but for some reason copying it is special. Checking it, it seems to have U+FE0F for some reason. U+FE0F is VARIATION SELECTOR-16 which [apparently forces emoji rendering](http://mts.io/2015/04/21/unicode-symbol-render-text-emoji/)...